api/v1alpha1/postgresqlhostcredentials_types.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// PostgreSQLHostCredentialsSpec defines the desired state of PostgreSQLHostCredentials
+type PostgreSQLHostCredentialsSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Host is the hostname of the PostgreSQL instance.
+	Host string `json:"host,omitempty"`
```

Comment:
I think we should make this a `ResourceVar` as well just to be future proof. I don't find it unlikely that maybe something like https://crossplane.io would provide a hostname to a cloud instance and with a `ResourceVar` we can reference it from a config map that crossplane would provide.

```suggestion
	Host ResourceVar `json:"host,omitempty"`
```